package.json

```diff
@@ -31,25 +31,28 @@
     "test": "npm run clean && npm run lint && npm run spec"
   },
   "dependencies": {
-    "asar": "^0.13.0",
-    "async": "^2.1.5",
-    "debug": "^2.6.3",
-    "fs-extra": "^2.1.2",
-    "glob": "^7.1.1",
+    "asar": "^0.14.0",
+    "async": "^2.6.0",
+    "debug": "^3.1.0",
+    "fs-extra": "^4.0.3",
+    "g": "^2.0.1",
```

Comment:
What's this?

Reply:
This is very weird... I updated the modules using `ncu -a`,  this `g` module shouldn't be there. I'll make sure to remove it.